### PR TITLE
feat: Use geolocation API to get the users location

### LINKED ISSUE #3: Using Geolocation API to get the users location
Using the Geolocation API to get the users location as the first weather results displayed, instead of just New York City.

Read more [here](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API).

### PR DESCRIPTION
**Pull request by AkhileshManda**
closes #3

### CODE REVIEW

**Comment by mrbazzan** [comment]
I'm checking it locally...

**Comment by sumana2001** [comment]
@mrbazzan You can check it in the deploy preview

**Comment by mrbazzan** [comment]
Ahh... yes. I almost started cloning it

Thanks @sumana2001

**Comment by mrbazzan** [comment]
> You can check it in the deploy preview

@sumana2001 Do we care about the little details like whitespace?

**Comment by sumana2001** [comment]
> @sumana2001 Do we care about the little details like whitespace?

It's completely on what you think. Even if you feel changes are not required, you can suggest whatever best practices you think are necessary while reviewing

File: src/App.js [hunk 1]
```diff
@@ -57,4 +81,5 @@ function App() {
   }
 }
 
+
```
Comment (mrbazzan) [inline src/App.js:84]: @AkhileshManda 
Is there a reason for this extra whitespace?

**Comment by joshi-kaushal** [comment]
We were implementing the same thing while resolving issue #5. It would be helpful if this PR gets merged sooner so we can work accordingly.

CC: @Pravallika-Myneni

**Comment by sumana2001** [comment]
> We were implementing the same thing while resolving issue #5. It would be helpful if this PR gets merged sooner so we can work accordingly.
> 
> CC: @Pravallika-Myneni

@mrbazzan @AkhileshManda please finish the PR asap

File: src/App.js (hunk 1 above)
Comment (AkhileshManda) [reply src/App.js]: No reason @mrbazzan , removed it with my latest commit

Comment (mrbazzan) [reply src/App.js]: Nice!

**Comment by mrbazzan** [comment]
> It would be helpful if this PR gets merged sooner so we can work accordingly.
> 

Sorry for the delay @joshi-kaushal

**Review by mrbazzan — APPROVED** [review]
LGTM

**Pull request merged**